.github/workflows/push.yml

```diff
@@ -87,3 +87,28 @@ jobs:
         run: docker build -t ghcr.io/policyengine/policyengine docker
       - name: Push container
         run: docker push ghcr.io/policyengine/policyengine
+  Publish:
+    name: Publish to PyPI
+    runs-on: ubuntu-latest
+    if: |
+      (github.repository == 'PolicyEngine/policyengine-api')
+      && (github.event.head_commit.message == 'Update PolicyEngine API')
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Setup Python
+        uses: actions/setup-python@v4
```

Comment:
We use `actions/setup-python@v5` elsewhere in this script. If this must be v4, could you comment that in here somewhere? Otherwise, could you fix at v5?